src/sortStudents.ts

```diff
@@ -1,16 +1,58 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function average(students: Student): number {
+  const avg: number = students.grades.reduce((a, b) => a + b);
```

Comment:
```suggestion
  const avg: number = students.grades.reduce((a, b) => a + b, 0);
```

don't forget about initial value